gradle.properties

```diff
@@ -28,24 +29,24 @@ junitVersion=4.13.2
 logbackVersion=1.2.7
 legacyConvertersVersion=5.0.0
 methvinDirectoryWatcherVersion=0.18.0
-micronautCacheVersion=3.5.0
-micronautRuntimeGroovyVersion=3.4.0
-micronautSpringVersion=4.5.1
-micronautVersion=3.10.4
+micronautCacheVersion=4.3.0
+micronautRuntimeGroovyVersion=4.3.0
+micronautSpringVersion=5.7.0
+micronautVersion=4.5.3
 mongodbJavaDriverVersion=4.11.0
 objenesisVersion=3.3
 scaffoldingCoreVersion=2.1.0
-servletApiVersion=4.0.1
+servletApiVersion=6.0.0
 slf4jVersion=1.7.36
 snakeyamlVersion=2.2
 spockVersion=2.3-groovy-4.0
-springBootVersion=2.7.18
+springBootVersion=3.2.6
```

Comment:
why not 3.3.1?

Reply:
From the previous discussions I was unsure if there is a problem with hibernate 5.6-jakarta in Boot 3.3 so I held off at upgrading to that until it can be tested.